Sarah_AlFadly/week_01/day_02/HW/variables/TheGeometrizer/main.js

```diff
@@ -0,0 +1,7 @@
+var radius = 10;
+
+var circumference = 3.14 * (10 * 2);
+var area = 3.14 * (10 ^ 2);
```

Comment:
Almost got it Sarah, remember power in JS is different, you can read about how to do power with JS [here](https://www.w3schools.com/jsref/jsref_pow.asp).